.profile.d/heroku-metrics-daemon.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # don't do anything if we don't have a metrics url.
-if [[ -z "$HEROKU_METRICS_URL" ]] || [[ "${DYNO}" = *run.* ]]; then
+if [[ -z "$HEROKU_METRICS_URL" ]] || [[ "${DYNO}" = run.* ]]; then
```

Comment:
should it be `run\.*` ? I'm not sure if the . is interpreted as a `.` or any single character.